public/js/selfoss-events-navigation.js

```diff
@@ -173,12 +170,13 @@ selfoss.events.navigation = function() {
                     $('#nav-mobile-settings').click();
                     
                 // refresh list
-                 selfoss.reloadList();
+                selfoss.sync(true);
```

Comment:
Refreshing sources could take some time. The point of the change is to let that run in the background and let the user continue reading and reload list ONLY if it is empty. If list has to be reloaded when refreshing is done even if items may still be displayed thus red by the user, I would rather revert 3071660c7c600aa68562f82c9e9db36ed7ca7867 .